cloudbuild.yaml

```diff
@@ -11,36 +11,27 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+timeout: 300s
+options:
+  machineType: 'N1_HIGHCPU_8'
 
 steps:
 
-# Run tests
+# Build all
```

Comment:
Seems like this step is no necessary. The binary is built in "Build an image" step.

Reply:
The build image step only builds specific targets. This is meant as a test that everything still compiles. I'll add a comment.